README.md

```diff
@@ -7,38 +7,61 @@
 [![contributers](https://img.shields.io/github/contributors/chichou/grapefruit)](https://github.com/ChiChou/Grapefruit/graphs/contributors)
 [![License](https://img.shields.io/github/license/chichou/grapefruit)](https://github.com/ChiChou/Grapefruit/blob/master/LICENSE)
 
+
 ## Get Started
+ 
+Grapefruit requires the following tools to build:
+
+1. [Node.js][node] `LTS`. For Apple Silicon, use Node.js `Current`.
+2. [Frida][frida] on computer.
```

Comment:
frida is not required on computer. It's going to be installed by `npm install` and this is misleading

Reply:
Ok, don't now about this 😅